docs/components/pagination.md

```
@@ -1,193 +1,273 @@
+import FwbPaginationExampleIcons from './pagination/examples/FwbPaginationExampleIcons.vue';
+import FwbPaginationExampleNavigation from './pagination/examples/FwbPaginationExampleNavigation.vue';
+import FwbPaginationExampleNavigationIcons from './pagination/examples/FwbPaginationExampleNavigationIcons.vue';
+import FwbPaginationExampleSlots from './pagination/examples/FwbPaginationExampleSlots.vue';
+import FwbPaginationExampleTable from './pagination/examples/FwbPaginationExampleTable.vue';
+import FwbPaginationExampleCustomLength from './pagination/examples/FwbPaginationExampleCustomLength.vue';
+import FwbPaginationExampleCustomLabels from './pagination/examples/FwbPaginationExampleCustomLabels.vue';
 </script>
+
 # Vue Pagination - Flowbite
-#### Use the Tailwind CSS pagination element to indicate a series of content across various pages based on multiple styles and sizes
+#### Use the Tailwind CSS pagination element to indicate a series of content across various pages based on multiple styles and sizes.
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)